doc/cli_vs_yum.rst

```diff
@@ -99,6 +94,12 @@ following will work::
 
 ``include`` directive name of [main] and Repo configuration is a more logical and better named counterpart of ``exclude`` in DNF.
 
+=======================================================================
+``include`` option has been removed
+=======================================================================
+
+The inclusion of external configuration files is no longer supported.
```

Comment:
I'd replace "external" with "remote" and drop the particle here:
> **Inclusion** of **remote** configuration files ...

Reply:
Since the option was removed completely should I change it to _Inclusion of configuration files..._ ?

Reply:
Yeah, maybe I'd go with:
> Inclusion of other configuration files in the main configuration file is no longer supported.